images-import/action.yml

```
@@ -15,16 +15,22 @@
 name: k3s Images Import
 description: Imports an image into the k3s cluster.
 inputs:
-  image:
-    description: The image to be imported to the k3s cluster.
+  images:
+    description: |-
+      The images to be imported to the k3s cluster. One image per line is
+      accepted.
     required: true
 outputs: {}
 runs:
   using: composite
   steps:
   - run: |-
-      set -o errexit -o pipefail
-      >&2 echo "Importing image ${{ inputs.image }}..."
-      docker save "${{ inputs.image }}" \
-        | sudo k3s ctr images import /dev/stdin
+      set -o errexit -o nounset -o pipefail
+
+      images=(
+        ${{ inputs.images }}
+      )
+
+      >&2 echo "Importing images ${images[@]}..."
```

```suggestion
      >&2 echo "Importing images ${images[*]}..."
```
It doesn't _actually_ matter, but here the expansion is used as a scalar (as in, a single string rather than a list of arguments).

---

Yes, either way, it gets interpolated into the string. In any case, I'll take the suggestion.